src/promptflow-evals/promptflow/evals/evaluators/_groundedness/_groundedness.py

```
@@ -75,7 +75,10 @@ def __call__(self, *, answer: str, context: str, **kwargs):
             raise ValueError("Both 'answer' and 'context' must be non-empty strings.")
 
         # Run the evaluation flow
-        llm_output = self._flow(answer=answer, context=context)
+        try:
+            llm_output = self._flow(answer=answer, context=context)
+        except BaseException as e:
+            print(e)
```

Why we eat the exception?

---

It was my debug code, removed.